index.html

```diff
@@ -78,196 +82,192 @@ <h3 class="contact-p">LET’S CONNECT</h3>
+              <div class="description-wraper work-child">
+                <h3 class="project-name">Uber</h3>
+                <img alt="dot" src="./assets/imgs/Counter.png" />
+                <p>Lead Developer</p>
+                <img alt="dot" src="./assets/imgs/Counter.png" />
+                <p>2018</p>
+              </div>
+              <p class="work-text work-child">
+                A smart assistant to make driving more safe, efficient, and fun
+                by unlocking your most expensive computer: your car.
+              </p>
+              <ul class="frameworks-list work-child">
+                <li class="framework">html</li>
+                <li class="framework">Ruby on rails</li>
+                <li class="framework">css</li>
+                <li class="framework">javaScript</li>
+              </ul>
+              <button type="button" class="project-btn work-child">
+                See Project
+              </button>
             </div>
-            <p class="work-text work-child">
-              A smart assistant to make driving more safe, efficient, and fun by
-              unlocking your most expensive computer: your car.
-            </p>
-            <ul class="frameworks-list work-child">
-              <li class="framework">html</li>
-              <li class="framework">Ruby on rails</li>
-              <li class="framework">css</li>
-              <li class="framework">javaScript</li>
-            </ul>
-            <button type="button" class="project-btn work-child">
-              See Project
-            </button>
           </div>
         </div>
       </section>
       <div id="container2">
-        <section id="about-me">
-          <div id="aboutme-desc">
-            <h2 class="work-child" id="ab-me">About Myself</h2>
-            <p class="work-child" id="ab-me-text">
-              Hello I’m a software developer! I can help you build a product ,
-              feature or website Look through some of my work and experience! If
-              you like what you see and have a project you need coded, don’t
-              hestiate to contact me.
-            </p>
-            <h3 class="contact-p work-child">LET’S CONNECT</h3>
-            <ul class="work-child icons2 icons">
-              <li>
-                <a target="blank" href="https://twitter.com/LisandroSeia"
-                  ><img alt="twitter" src="./assets/imgs/twitter.png"
-                /></a>
-              </li>
-
-              <li>
-                <a
-                  target="blank"
-                  href="https://www.linkedin.com/in/lisandro-seia-295120225/"
-                  ><img alt="linkedin" src="./assets/imgs/linkedin.png"
-                /></a>
-              </li>
-
-              <li>
-                <a href="#"
-                  ><img alt="medium" src="./assets/imgs/medium.png"
-                /></a>
-              </li>
+        <section id="about-me-section">
+          <div id="about-me">
```

Comment:
- [ ] Please kindly match your  design with the one on figma, currently, about-me section is different:

Figma  | Yours
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/144489891-72358c84-72dc-4d55-9e89-1b1787ae8986.png)  | ![image](https://user-images.githubusercontent.com/77038610/144489950-f7507bd5-1894-4f6f-b813-be5ad76d7fd0.png)

You can start from decreasing padding to give more space for the left section 😉 

- [ ] After `About Myself `header, there should be only 4 to 5 lines of text describing your experience. In your case however, it is 7 😮